dev-tools/mage/manifest/manifest.go

```diff
@@ -99,6 +101,13 @@ func DownloadComponentsFromManifest(manifest string, platforms []string, platfor
 	}
 	projects := manifestResponse.Projects
 
+	// If using new Independent Agent version, just use the first X.Y.Z part
+	majorMinorPatchVersion := manifestResponse.Version
```

Comment:
nit, for parsing the version we can use the https://github.com/elastic/elastic-agent/tree/main/pkg/version package and extract the part of the version we want.